ramls/data-import-converter-storage.raml

```diff
@@ -218,8 +218,30 @@ resourceTypes:
         exampleCollection: !include raml-storage/examples/mod-data-import-converter-storage/profileAssociationCollection.sample
         exampleItem: !include raml-storage/examples/mod-data-import-converter-storage/profileAssociation.sample
     post:
+      queryParameters:
+        master:
+          description: It needs to identify master profile type in association
+          required: true
+          type: string
+          example: "JOB_PROFILE, ACTION_PROFILE, MAPPING_PROFILE, MATCH_PROFILE"
+        detail:
+          description: It needs to identify master profile type in association
```

Comment:
It needs to identify **MASTER** profile type in association ?
I think we should specify here the valid pairs of profiles because when somebody call it with: 
master = MATCH_PROFILE
detail= JOB_PROFILE 
he/she get an exception